src/main/DAL/MetaQuestion/MetaQuestionRepository.js

```diff
@@ -44,12 +80,78 @@ class MetaQuestionRepository {
      * @returns The updated question.
      */
     async addAnswersToQuestion(qId, answers) {
-        await answers.forEach(answer => {
+        answers.forEach(answer => {
```

Comment:
I think you are the one who told me to change from answers to keys

Reply:
`correctAnswers` changed to `keys`
$Answers = Keys \cup Distractors$
Why do we work with the superset `Answers` then?
An answer might be wrongly tagged as a `key` while in actuality be wrong in relation to the `MetaQuestion` it is connected to: 
- If we work with the `Answer` model we simply switch the tag from `key` to `distractor. 
- If we work with the `Key` and `Distractor` model, in order to switch the tagging we will need to remove the instance from the `Keys` table and create the instance as a row in the `Distractors` table.